nuopc.runconfig

```diff
@@ -300,7 +300,7 @@ CLOCK_attributes::
      tprof_n = -999
      tprof_option = never
      tprof_ymd = -999
-     wav_cpl_dt = 3600
+     wav_cpl_dt = 99999       #not used
```

Comment:
```suggestion
     wav_cpl_dt = 99999 #not used
```

Reply:
Resolved in this commit [5899c1c](https://github.com/COSIMA/MOM6-CICE6/pull/56/commits/5899c1c4b8c0c972a5fb526147031e1fea3f5fd5)